bin/journey-test.sh

```diff
@@ -233,15 +233,15 @@ solve_all_exercises() {
 
     pushd ${exercism_exercises_dir}/${TRACK}/${exercise}
     # Check that tests compile before we strip @Ignore annotations
-    "$EXECPATH"/gradlew compileTestJava
+    "$EXECPATH"/gradlew compileTestJava --configure-on-demand --parallel
```

Comment:
So rerunning them seemed to not show much difference...
[With configure-on-demand](http://logans.me/profile/profile-2017-07-07-22-19-31.html)
[Without](http://logans.me/profile/profile-2017-07-07-22-24-43.html)